mc-oblivious-ram/src/evictor/mod.rs

```diff
@@ -22,28 +23,113 @@ use aligned_cmov::typenum::PartialDiv;
 
 use aligned_cmov::ArrayLength;
 
+use rand_core::CryptoRng;
+
 use crate::path_oram::{BranchCheckout, MetaSize};
+use rand_core::RngCore;
```

Comment:
:upside_down_face: 
I wouldn't suggest as part of this PR, but the newlines between all of the use statements is preventing cargo fmt from consolidating
```suggestion
use rand_core::{CryptoRng, RngCore};

use crate::path_oram::{BranchCheckout, MetaSize};
```